jax/experimental/jax2tf/tests/primitives_test.py

```diff
@@ -284,52 +282,51 @@ def test_select_and_gather_add(self, harness: primitive_harness.Harness):
   def test_unary_elementwise(self, harness: primitive_harness.Harness):
     dtype = harness.params["dtype"]
     lax_name = harness.params["lax_name"]
-    if dtype is dtypes.bfloat16:
-      raise unittest.SkipTest("bfloat16 not implemented")
+    # TODO(bchetioui): do they have bfloat16 support, though?
     if lax_name in ("sinh", "cosh", "atanh", "asinh", "acosh") and dtype is np.float16:
       raise unittest.SkipTest("b/158006398: float16 support is missing from '%s' TF kernel" % lax_name)
     arg, = harness.dyn_args_maker(self.rng())
     custom_assert = None
     if lax_name == "digamma":
       # TODO(necula): fix bug with digamma/f32 on TPU
-      if harness.params["dtype"] is np.float32 and jtu.device_under_test() == "tpu":
+      if dtype is np.float32 and jtu.device_under_test() == "tpu":
```

Comment:
Should we merge the cases of float32 and float16?

Reply:
Done